pkg/controller/attach_detach.go

```diff
@@ -741,45 +735,47 @@ func (r *ReconcileAttachDetach) recoverAzVolumeAttachment(ctx context.Context, r
 		wg.Add(1)
 		go func(azv azdiskv1beta2.AzVolumeAttachment, azvMap *sync.Map) {
 			defer wg.Done()
+			var publishCtx map[string]string
+			if lun, _, err := r.cloudDiskAttacher.GetDiskLun(azv.Spec.VolumeName, azv.Spec.VolumeID, types.NodeName(azv.Spec.NodeName)); err == nil {
+				publishCtx = map[string]string{"LUN": strconv.Itoa(int(lun))}
```

Comment:
Use `azureconstants.LUN` instead of `"LUN"` here.

Reply:
Then here should be updated as well.
https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/509d944d1b21e66112bc0aaf38047131856f0397/pkg/provisioner/cloudprovisioner.go#L475